components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -531,7 +537,7 @@ function createInviteChannelButton(channel) {
                     {(title) => (
                         <i
                             className='fa fa-user-plus'
-                            title={title}
+                            title={title as string}
```

Comment:
Same question as before, about adding the type to the parameter instead of type casting (if possible).